WebEngine/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved. Use of this source
+# code is governed by a BSD-style license that can be found in the LICENSE file.
```

Comment:
The license headers of the other CMake files have newlines in the same places as the C++ files.

Reply:
Fixed.